src/NewsService.Models.Dto/Models/FindNewsParams.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace LT.DigitalOffice.NewsService.Models.Dto.Models
+{
+    public class FindNewsParams
+    {
+        public Guid? AuthorId { get; set; }
+        public Guid? DepartmentId { get; set; }
+        public string Pseudonym { get; set; }
```

Comment:
Why do you rename this field from AuthorName -> Pseudonym? for convenience?

Reply:
in this property it is necessary to save not the name but the pseudonym, the author's name is obtained from the userSevravise.